kafka-ui-react-app/src/components/Brokers/Broker/__test__/Broker.spec.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { render } from 'lib/testHelpers';
+import { screen, waitFor } from '@testing-library/dom';
+import { Route } from 'react-router-dom';
+import { clusterBrokerPath } from 'lib/paths';
+import fetchMock from 'fetch-mock';
+import {
+  clusterStatsPayload,
+  clusterStatsPayloadBroker,
+} from 'redux/reducers/brokers/__test__/fixtures';
+import { act } from '@testing-library/react';
+import Broker from 'components/Brokers/Broker/Broker';
+
+describe('Broker Component', () => {
+  afterEach(() => fetchMock.reset());
+
+  const clusterName = 'local';
+  const brokerId = 1;
+
+  const renderComponent = () =>
+    render(
+      <Route path={clusterBrokerPath(':clusterName', ':brokerId')}>
+        <Broker />
+      </Route>,
+      {
+        pathname: clusterBrokerPath(clusterName, brokerId),
+      }
+    );
+
+  describe('Broker', () => {
+    let fetchBrokersMock: fetchMock.FetchMockStatic;
+    const fetchStatsUrl = `/api/clusters/${clusterName}/stats`;
+    const fetchBrokerMockUrl = `/api/clusters/${clusterName}/brokers/logdirs?broker=${brokerId}`;
+
+    beforeEach(() => {
+      fetchBrokersMock = fetchMock.getOnce(
+        `/api/clusters/${clusterName}/brokers/${brokerId}`,
+        clusterStatsPayload
+      );
+    });
+
+    it('renders', async () => {
+      const fetchStatsMock = fetchMock.getOnce(
+        fetchStatsUrl,
+        clusterStatsPayload
+      );
+      const fetchBrokerMock = fetchMock.getOnce(
+        `/api/clusters/${clusterName}/brokers/logdirs?broker=${brokerId}`,
+        clusterStatsPayloadBroker
+      );
+
+      await act(() => {
+        renderComponent();
+      });
+
+      await waitFor(() => expect(fetchStatsMock.called()).toBeTruthy());
+      await waitFor(() => expect(fetchBrokersMock.called()).toBeTruthy());
+      await waitFor(() => expect(fetchBrokerMock.called()).toBeTruthy());
+
+      expect(screen.getByRole('table')).toBeInTheDocument();
+      const rows = screen.getAllByRole('row');
+      expect(rows.length).toEqual(2);
+    });
+
+    it('shows warning when broker not found', async () => {
+      const fetchStatsMock = fetchMock.getOnce(
+        fetchStatsUrl,
+        clusterStatsPayload
+      );
+      const fetchBrokerMock = fetchMock.getOnce(fetchBrokerMockUrl, []);
+      await act(() => {
+        renderComponent();
+      });
+      await waitFor(() => expect(fetchStatsMock.called()).toBeTruthy());
+      await waitFor(() => expect(fetchBrokersMock.called()).toBeTruthy());
+      await waitFor(() => expect(fetchBrokerMock.called()).toBeTruthy());
+
+      expect(screen.getByRole('table')).toBeInTheDocument();
+      const rows = screen.getAllByRole('row');
+      expect(rows.length).toEqual(2);
```

Comment:
there is no run same expects in every case
```
await waitFor(() => expect(fetchStatsMock.called()).toBeTruthy());
      await waitFor(() => expect(fetchBrokersMock.called()).toBeTruthy());
      await waitFor(() => expect(fetchBrokerMock.called()).toBeTruthy());

      expect(screen.getByRole('table')).toBeInTheDocument();
      const rows = screen.getAllByRole('row');
      expect(rows.length).toEqual(2);
      ```
      You have already tested it in previous case